docs/source/_toctree.yml

```diff
@@ -125,6 +125,11 @@
     isExpanded: false
   title: BetterTransformer
   isExpanded: false
+- sections:
+  - local: optimization_toolbox/usage_guides/quantization
+    title: GPTQ quantization
+  title: Optimization toolbox
```

Comment:
Not sure about this title as many things in Optimum are about optimization :sweat_smile: 
I would just name it "GPTQ" for now, or maybe better "LLM quantization". WDYT?

Reply:
Yes ! LLM quantization sounds good =)